Presentation.Web/app/components/user-notification/user-notification-modal.controller.ts

```diff
@@ -7,6 +7,7 @@
         mainGrid; // We don't set the grid. It is being set by the options that we define. But we need access to this in order to manually force a refresh of the data.
 
         static $inject: Array<string> = [
+            "$scope",
```

Comment:
Det er en controller.. kan du ikke bare sætte den på VM i stedet for at bruge non-controller tilgangen?